config/sidekiq.yml

```diff
@@ -4,6 +4,5 @@
   - priority_mailers
   - mailers
   - default
-  - big_query
```

Comment:
Do we need to remove this queue via sidekiq web as well? Or just removing here will do the trick?

Reply:
I believe removing it here should be enough; we can check the UI afterwards though to make sure it all looks right 👍